src/app/messaging/component/message-window/message-window.component.ts

```diff
@@ -55,6 +61,20 @@ export class MessageWindowComponent implements OnInit, OnDestroy {
           (messages => this.data = messages);
   }
 
+  sendMessage(){
+    let newMessage=new CreateMessage
+    newMessage.content=this.messageString
+    this.messagesService.sendMessage(newMessage,this.conversationID)
+  }
+
+  doTextareaValueChange(ev:any) {
```

Comment:
Wouldn't ngModel accomplish this for us without a whole method?

Reply:
done.